src/riscv-integration.adoc

```diff
@@ -545,7 +545,9 @@ include::img/mscratchreg.edn[]
 ==== Machine Scratch Register Capability (mscratchc)
 
 The <<mscratchc>> register is an extension to <<mscratch>> that is able to hold
-a capability. Its reset value is the <<null-cap>> capability.
+a capability.
+
+{TAG_RESET_CSR}
```

Comment:
Could you please also fix the text of these other CSRs?
* mepcc sepcc: Currently states that the reset value is NULL
* sscratchc

Reply:
yes ok - thanks for checking